ember-resources/src/util/function.ts

```diff
@@ -1,15 +1,22 @@
 import { tracked } from '@glimmer/tracking';
```

Comment:
why so many formatting changes here? can you revert these? makes the diff a bit noisy

Reply:
Done, prettier format on save :sweat: